qa/os/src/test/java/org/elasticsearch/packaging/test/ConfigurationTests.java

```diff
@@ -22,22 +22,44 @@
 import org.apache.http.client.fluent.Request;
 import org.elasticsearch.packaging.util.FileUtils;
 import org.elasticsearch.packaging.util.Platforms;
+import org.junit.After;
 import org.junit.Before;
 
+import java.nio.file.Files;
+import java.util.List;
+
+import static java.nio.file.StandardOpenOption.APPEND;
+import static java.nio.file.StandardOpenOption.CREATE;
 import static org.elasticsearch.packaging.util.FileUtils.append;
 import static org.elasticsearch.packaging.util.ServerUtils.makeRequest;
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assume.assumeFalse;
 
 public class ConfigurationTests extends PackagingTestCase {
 
     @Before
-    public void filterDistros() {
+    public void filterDistrosAndInstall() throws Exception {
         assumeFalse("no docker", distribution.isDocker());
+        install();
```

Comment:
I don't think install should be in a Before. This means we would reinstall before each test, which is costly. The packaging tests are generally written to cleanup after themselves, so deleting the installation should not be necessary?

Reply:
I'm getting errors when I don't reinstall. I'll dig on it.

Reply:
I found the problem. I am going to fix it in a different PR because it's unrelated to this ticket and it ought to be fixed on master as well.